WWDC/SessionsTableViewController.swift

```diff
@@ -497,6 +501,17 @@ class SessionsTableViewController: NSViewController {
     }
 }
 
+extension Session {
+
+    var isWatched: Bool {
```

Comment:
In your opinion, where should this live @insidegui ? The `Constants` are part of WWDC, while `Session` is part of ConfCore. So it feels fine as an extension defined in WWDC but could probably use a better home.

Reply:
Interesting question. I think we can leave it as is by now, but maybe some constants should be moved into ConfCore (or have some entity in ConfCore that can hold these types of configurations).